pkg/controller/androidvariant/androidvariant_controller.go

```diff
@@ -83,6 +84,17 @@ func (r *ReconcileAndroidVariant) Reconcile(request reconcile.Request) (reconcil
 		return reconcile.Result{}, err
 	}
 
+	// Check if the namespace of the cr is in the APP_NAMESPACES environment variable provided to the operator or if it's in the operator namespace
+	operatorNamespace, err := k8sutil.GetOperatorNamespace()
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+	is := util.IsValidAppNamespace(instance.Namespace, operatorNamespace, instance.Name)
+	if !is {
+		reqLogger.Info("The app cr %s was created in a namespace which is not present in the APP_NAMESPACES environment variable provided to the operator or is not the in the operator namespace", instance.Name)
```

Comment:
```suggestion
		reqLogger.Info(fmt.Sprintf("The app cr %s was created in a namespace which is not present in the APP_NAMESPACES environment variable provided to the operator or is not the in the operator namespace", instance.Name))
```

This should fix the error that `reqLogger.Info` doesn't have the correct number of parameters, and also populates the `%s` in the format string as expected.